app/views/search/partials/filters/_workplace_financial_advice.html.erb

```diff
@@ -1,22 +1,22 @@
-<fieldset class="search-filter__pension_pot">
+<fieldset class="search-filter__investment_sizes">
   <legend class="search-filter__label">
     <%= t('search.filter.workplace_financial_advice_flag.heading') %>
   </legend>
 
   <div class="form__group-item">
-    <%= f.check_box 'setting_up_workplace_pension_flag', class: "form__group-input t-setting_up_workplace_pension search-filter__checkbox" %>
+    <%= f.check_box 'setting_up_workplace_pension_flag', class: "form__group-input t-setting_up_workplace_pension search-filter__checkbox", name: 'search_form[filters[setting_up_workplace_pension_flag]]' %>
```

Comment:
Is it worth putting this markup on several lines?

Reply:
Thanks, good call. It has been addressed now.